src/ch/ehi/ili2db/fromili/FromIliRecordConverter.java

```diff
@@ -194,7 +196,28 @@ public void generateTable(ViewableWrapper def,int pass)
 		if(base==null && !def.isSecondaryTable()){
 			if(createTypeDiscriminator || def.includesMultipleTypes()){
 				  dbCol=createSqlTypeCol(DbNames.T_TYPE_COL);
-				  dbTable.addColumn(dbCol);
+
+                  ArrayList<String> extensions = new ArrayList<String>();
+                  for (Object o : def.getViewable().getExtensions()){
+                      extensions.add(ili2sqlName.mapIliClassDef((Viewable) o));
+                  }
+
+                  // Add t_type possible values to meta-info table
+                  metaInfo.setColumnInfo(dbTable.getName().getName(),
+                                         DbNames.T_TYPE_COL,
+                                         DbExtMetaInfo.TAG_COL_TYPES,
+                                         extensions.toString());
+
```

Comment:
I add the possible values into the meta info table like this: `[class_a, class_b, class_c]`
Is that okay or do I need to format it in another way?

Reply:
please use JSON syntax

Reply:
done